packages/@aws-cdk-testing/framework-integ/test/aws-s3/test/integ.bucket-block-public-access.ts

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+import * as cdk from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as cxapi from 'aws-cdk-lib/cx-api';
+
+const myFeatureFlag = { [cxapi.S3_BUCKET_DEFAULT_BLOCK_PUBLIC_ACCESS_PROPERTIES_TO_TRUE]: true };
+
+const app = new cdk.App({
+  context: myFeatureFlag,
+});
+
+const stack = new cdk.Stack(app, 'aws-cdk-s3');
+
+// Bucket with default setting for `blockPublicAccess`
+new s3.Bucket(stack, 'BucketWithDefaultAccess', {});
+
+// Bucket with explicit setting for `blockPublicAccess`
+new s3.Bucket(stack, 'BucketWithExplicitBlockAccess', {
+  blockPublicAccess: new s3.BlockPublicAccess({
+    blockPublicAcls: true,
+    ignorePublicAcls: true,
+    blockPublicPolicy: true,
+    restrictPublicBuckets: true,
+  }),
+});
+
+// Bucket with partial setting for `blockPublicAccess`
+new s3.Bucket(stack, 'BucketWithPartialBlockAccess', {
+  blockPublicAccess: new s3.BlockPublicAccess({
+    blockPublicPolicy: false,
+    restrictPublicBuckets: false,
+  }),
+});
+
+new IntegTest(app, 'cdk-integ-s3-bucket-block-public-access', {
```

Comment:
could we add an assertion to this test to check that the `BucketWithPartialBlockAccess` sets the unspecified properties to `true`? From [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-publicaccessblockconfiguration.html#aws-properties-s3-bucket-publicaccessblockconfiguration-properties) it seems like 
```
PUT Bucket ACL and PUT Object ACL calls fail if the specified ACL is public.

PUT Object calls fail if the request includes a public ACL.

PUT Bucket calls fail if the request includes a public ACL.
```